packages/application-components/src/components/custom-views/custom-views-selector/custom-views-selector.spec.tsx

```diff
@@ -0,0 +1,104 @@
+import { graphql } from 'msw';
+import { setupServer } from 'msw/node';
+import {
+  screen,
+  renderApp,
+} from '@commercetools-frontend/application-shell/test-utils';
```

Comment:
ℹ️ We can use the application shell test-utils here, no need to rebuild the test setup.